frontend/src/auth/auth.js

```diff
@@ -1,6 +1,6 @@
 import Vue from 'vue'
```

Comment:
Du kannst diese Datei in index.js umbenennen, dann kann man weiterhin `import '@/auth'` schreiben (statt `import '@/auth/auth'`)

Reply:
Okay, ich hatte gedacht, so sei es verständlicher was das Testfile testet.

Reply:
Meinst du das?
https://alligator.io/react/index-js-public-interfaces/#using-indexjs

Dann würde ich das auch für den Store machen.

Reply:
@carlobeltrame 
Meinst du das?
https://alligator.io/react/index-js-public-interfaces/#using-indexjs

Dann würde ich das auch für den Store machen.

Reply:
Ja stimmt, den API-Store hattest du umbenannt. Fände ich auch dort wieder gut index.js zu haben.